packages/realm/binding/jsi/CMakeLists.txt

```diff
@@ -1,19 +0,0 @@
-# Using Node.js to resolve the path to the "react-native" package.
```

Comment:
Our jsi target is removed, because the Android target and iOS Podspec points directly to `jsi_init.cpp`.